aio/content/examples/attribute-binding/src/app/comp-with-host-binding.component.ts

```diff
@@ -2,18 +2,17 @@ import { Component, HostBinding } from '@angular/core';
 
 @Component({
   selector: 'comp-with-host-binding',
-  template: 'I am a component!',
+  template: `I am a component!`,
```

Comment:
I think we should revert this change as it's more common to use single quotes for strings without JS interpolations:
```suggestion
  template: 'I am a component!',
```